### PR TITLE
Find min/max prices when price ranges are entered in the "Cost" field

### PR DESCRIPTION
**Pull request by borkweb**
See: https://central.tri.be/issues/36518 and https://central.tri.be/issues/36620

Note: this was already code reviewed in https://github.com/moderntribe/the-events-calendar/pull/82 and this is just pointing those changes at `develop` rather than `release/119`.

### CODE REVIEW

File: src/Tribe/Cost_Utils.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+	/**
+	 * fetches all event costs from the database
+	 *
+	 * @return array
+	 */
+	public function get_all_costs() {
+		global $wpdb;
+
+		$costs = $wpdb->get_col( "SELECT meta_value FROM {$wpdb->postmeta} WHERE meta_key = '_EventCost'" );
```
Comment (zbtirrell) [inline src/Tribe/Cost_Utils.php]: Why not use a native WordPress function here to fetch this?

Comment (borkweb) [inline src/Tribe/Cost_Utils.php]: This isn't fetching the value for a single post, instead it is fetching it for _all_ posts.

Comment (bordoni) [inline src/Tribe/Cost_Utils.php]: That is for sure missing on WordPress.

**Comment by borkweb** [comment]
This passed QA. Merging!

**Pull request merged**